Dockerfile.linux-build

```
@@ -0,0 +1,83 @@
+COPY canine-chain /workspace/canine-chain
+COPY cometbft-sec-tachyon /workspace/cometbft-sec-tachyon
+
+# Set working directory to canine-chain
+WORKDIR /workspace/canine-chain
+
+# Download dependencies first to locate libwasmvm.so
+RUN go mod download
+
+# Find and copy libwasmvm.so to a standard location
+# Select the correct architecture based on GOARCH
+RUN if [ "$GOARCH" = "amd64" ]; then \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)